unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PoolManager/Components/ReleaseParticlesOnFinish.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+namespace DCL
+{
+    public class ReleaseParticlesOnFinish : MonoBehaviour
+    {
+        private const int INDEX_AMOUNT = 10;
```

Comment:
i would rename to: `FRAME_RATE_CHECK`

Reply:
@lorux0 I am not sure about the proposed change, as is not related to any rate and I am using a system similar to the AI update behaviors for checking only on the index for the calculation.
I can try to find a better name though!

Reply:
ok!